packages/expo-updates/ios/EXUpdates/EXUpdatesService.m

```diff
@@ -7,17 +7,25 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#define SUPPRESS_EXPO_UPDATES_SERVICE
```

Comment:
we probably don't want this here, correct? otherwise this macro will be defined in bare projects where we don't want it. (or maybe I'm misunderstanding exactly what `#define` does?)